### ISSUE TITLE
Publish a new revision of the core specifications for public review

### ISSUE THREAD

**Issue by jamsden** [oslc-op/oslc-admin#30]
The Core specifications have the new conformance section added. This is significant enough of a change to warrant a new revision for public review. This can also include conversion to Project Specification format.

**Comment by berezovskyi** [comment]
I agree. I have updated them to render under ReSpec 2.1.6, I guess. I was thinking we should publish a PSD of RM or Core once the CM is published (today, hopefully).
